test/facade_converter_test.ts

```diff
@@ -182,7 +182,10 @@ external List<String> /*ReadonlyArray<String>*/ f();`);
 
   describe('error detection', () => {
     it('supports imports', () => {
-      expectWithTypes(`import {X} from "other";\nlet x:X;`).to.equal(`import "other.dart" show X;
+      expectWithTypes(`
+import {X} from "other";
+declare let x:X;
+`).to.equal(`import "../../other.dart" show X;
```

Comment:
why is other interpreted as being two directories up? That seems a bit odd.

Reply:
other.ts has a fake location within the top level directory whereas the test code has a fake location at 'demo/some/main.ts' 

https://github.com/dart-lang/js_facade_gen/blob/05d25aa6546b24e31dc8d25cba4e425ed1ffaf84/test/facade_converter_test.ts#L4
https://github.com/dart-lang/js_facade_gen/blob/05d25aa6546b24e31dc8d25cba4e425ed1ffaf84/test/test_support.ts#L74

It would be ideal to place other at 'demo/some/other.ts', but for some reason import {X} from 'other' doesn't work if I do that. That import structure does work outside of tests though, I think it's just a problem with the CompilerHost.

Reply:
That is fine. Go ahead and just document in the test why the output looks weird so the next person looking at it knows why that output is a good not a bad things.

Reply:
Will do